recipes/_packaging.rb

```diff
@@ -39,6 +39,9 @@
   package 'zlib-devel'
   # EL 7 split rpm-sign into its own package:  http://cholla.mmto.org/computers/linux/rpm/signing.html
   package 'rpm-sign' if node['platform_version'].satisfies?('>= 7')
+  # EL 6 and later consider glibc-static optional: https://access.redhat.com/solutions/33868
+  # The libhugetlbfs build (analytics) requires it. 
+  package 'glibc-static' if node['platform_version'].satisfies?('>= 6')
```

Comment:
@schisamo and I talked in standup, and we're good.  For everyone else, this hasn't come up in CI because the `runit` cookbook installs `glibc-static`:  https://github.com/hw-cookbooks/runit/blob/develop/recipes/default.rb#L54-55